cmd/runtimetest/main.go

```diff
@@ -191,6 +191,20 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateROPaths(spec *rspec.Spec) error {
+	fmt.Println("validating readonlyPaths")
+	for _, v := range spec.Linux.ReadonlyPaths {
+		fi, err := os.Stat(v)
```

Comment:
@wking  The spec says "`readonlyPaths will set the provided paths as readonly inside the container.`"
It seems not to remount readonlyPaths and just change their permisson. If this is true, I think current implementation is OK.